pkgs/xsoulspace_monetization_rustore/Makefile

```
@@ -0,0 +1,5 @@
+publish-dry:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)